api/v1alpha/framework/cypress.schema.json

```diff
@@ -60,6 +60,27 @@
         },
         "version": {
           "$ref": "../subschema/common.schema.json#/definitions/version"
+        },
+        "reporters": {
+          "description": "Set of reporter to use.",
```

Comment:
If it's not a built-in reporter, the user still has to add the appropriate package to the `npm.packages` category, right? That may not be immediately obvious and we should probably include that in the description 🤔

Reply:
Correct ! I'll update that message !